### PR TITLE
Prepares moments for API data

### PR DESCRIPTION
**Pull request by austinpray**
Missing:

- [x] transcripts
- [ ] body notes

### CODE REVIEW

File: src/utils/dummyData.js [hunk 1]
```diff
@@ -1,29 +1,45 @@
-export const dummyNotes = [
-  {id: 1, title: "We landed on the moon!", text: "We did that!"},
-  {id: 2, title: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.", text: "We did that!", image: "http://placehold.it/400x300"},
-  {id: 3, title: "Pellentesque dictum at libero in vulputate.", text: "こんにちは！お元気ですか？"},
-  {id: 4, title: "Nulla lacinia lacinia dolor eu sagittis.", text: "バズ・ライトイヤー"},
-  {id: 5, title: "test text", text: "In eros ante, facilisis vitae augue vitae, aliquam suscipit arcu!"},
-  {id: 6, title: "Praesent id justo vitae nunc ultricies egestas.", text: "We did that!"}
-];
+const momentTemplate = {
```
Comment (austinpray) [inline src/utils/dummyData.js]: @VictorK1902 might be useful to look at

File: src/utils/dummyData.js [hunk 2]
```diff
@@ -1,29 +1,45 @@
...
+  mission: {
+    id: 1,
+    title: "Apollo 11",
+    length: 7.031e8
+  },
+  startSlice: 7.031e8 / 2,
+  endSlice: 6e8,
+  audioUrl: "https://aqueous-garden-9236.herokuapp.com/stream.mp3",
+  transcripts: [
+    {id: 1, name: "Niel", text: "One small step for man."},
```
Comment (JamieCrisman) [inline src/utils/dummyData.js]: make sure this matches the transcript model

**Comment by JamieCrisman** [comment]
transcripts are essentially done with the addition that they need to set the audio time on click

**Comment by austinpray** [comment]
@JamieCrisman merge this?

**Comment by JamieCrisman** [comment]
yeah this is the one. just because a lot of the core stuff we're doing from now on is in this branch

**Pull request merged**